Makefile

```diff
@@ -35,6 +35,9 @@ imgs_server_dest := $(imgs_source:%=build/server/%)
 
 raven_source := $(shell node -e 'console.log(require.resolve("raven-js/dist/raven.js"))')
 
+l10n_source := $(wildcard locales/*/*.properties)
+l10n_dest := $(l10n_source:%/*.properties=addon/webextension/_locales/%/messages.json)
```

Comment:
I was thinking we'd eventually have multiple properties files in this directory, hence the double wildcard. However, now that I think about it, we would only ever want to package up the addon's strings in the addon...so I'll update this to grab `$(wildcard locales/*/webextension.properties)`